calico-enterprise/networking/configuring/node-local-dns-cache.mdx

```diff
@@ -0,0 +1,103 @@
+---
+description: Improve cluster DNS performance with NodeLocal DNSCache
+---
+
+# Improve cluster DNS performance with NodeLocal DNSCache
+
+## Big picture
+
+NodeLocal DNSCache deploys `node-local-dns` pods as hostnetwork on all cluster nodes to improve DNS perfomance.
+This means the route for DNS traffic is changing and new hops are being introduced in your network when enabling this feature.
+Thus, you need appropriate policies in place to allow for this new traffic in your cluster.
+
+After deploying NodeLocal DNSCache to your {{prodname}} cluster, you need to add a policy for `kube-dns` to allow ingress
```

Comment:
I'd suggest not including any steps or actions in the "big picture" section.  This should go further down as part of the list of steps.  The step exists later on anyway.